<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
    "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">

<html lang="en" xmlns="http://www.w3.org/1999/xhtml" xml:lang="en">
<head>
  <meta name="generator" content="WebStrom v2016.3.1 for Linux, see www.w3.org" />

  <title>About</title>
  <meta http-equiv="Content-Type" content="text/html; charset=us-ascii" />
  <link rel="stylesheet" href="includes/stylesheet.css" type="text/css" />
</head>

<body class="main">
  <!-- MAIN TABLE -->

  <table border="0" cellspacing="2" cellpadding="2">
    <tbody>
      <tr>
        <td class="links">
          <a href="index.html"><img src="includes/logo.png" alt="SMS logo" width="144" height="56" border="0" /></a> <!-- INSERT LINKS -->
    <div class="category">
    Sequence Analysis
  </div>

  <div class="program">
    <a href="codon_plot.html" onmouseover="self.status='Plots codon frequency (according to the codon table you enter) for each codon in a DNA sequence.'; return true;" onmouseout="self.status=' '; return true;">-Codon Plot</a><br />
    <a href="codon_usage.html" onmouseover="self.status='Returns a standard codon usage table.'; return true;" onmouseout="self.status=' '; return true;">-Codon Usage</a><br />
    <a href="cpg_islands.html" onmouseover="self.status='Returns a list of potential CpG islands.'; return true;" onmouseout="self.status=' '; return true;">-CpG Islands</a><br />
    <a href="dna_mw.html" onmouseover="self.status='Calculates the molecular weight of DNA sequences.'; return true;" onmouseout="self.status=' '; return true;">-DNA Molecular Weight</a><br />
    <a href="dna_pattern.html" onmouseover="self.status='Returns positions of the patterns you enter.'; return true;" onmouseout="self.status=' '; return true;">-DNA Pattern Find</a><br />
    <a href="dna_stats.html" onmouseover="self.status='Returns basic sequence statistics.'; return true;" onmouseout="self.status=' '; return true;">-DNA Stats</a><br />
    <a href="fuzzy_search_dna.html" onmouseover="self.status='Returns sequences that are identical or similar to a query sequence.'; return true;" onmouseout="self.status=' '; return true;">-Fuzzy Search DNA</a><br />
    <a href="fuzzy_search_protein.html" onmouseover="self.status='Returns sequences that are identical or similar to a query sequence.'; return true;" onmouseout="self.status=' '; return true;">-Fuzzy Search Protein</a><br />
    <a href="ident_sim.html" onmouseover="self.status='Accepts aligned sequences in FASTA format and calculates the identity and similarity of each sequence pair.'; return true;" onmouseout="self.status=' '; return true;">-Ident and Sim</a><br />
    <a href="multi_rev_trans.html" onmouseover="self.status='Can be used to predict a DNA sequence in another species using a protein sequence alignment.'; return true;" onmouseout="self.status=' '; return true;">-Multi Rev Trans</a><br />
    <a href="mutate_for_digest.html" onmouseover="self.status='Finds DNA sequences that can easily be converted to a restriction site.'; return true;" onmouseout="self.status=' '; return true;">-Mutate for Digest</a><br />
    <a href="orf_find.html" onmouseover="self.status='Determines the positions of open reading frames.'; return true;" onmouseout="self.status=' '; return true;">-ORF Finder</a><br />
    <a href="pairwise_align_codons.html" onmouseover="self.status='Returns the optimal global alignment for two coding DNA sequences.'; return true;" onmouseout="self.status=' '; return true;">-Pairwise Align Codons</a><br />
    <a href="pairwise_align_dna.html" onmouseover="self.status='Returns the optimal global alignment for two DNA sequences.'; return true;" onmouseout="self.status=' '; return true;">-Pairwise Align DNA</a><br />
    <a href="pairwise_align_protein.html" onmouseover="self.status='Returns the optimal global alignment for two protein sequences.'; return true;" onmouseout="self.status=' '; return true;">-Pairwise Align Protein</a><br />
    <a href="pcr_primer_stats.html" onmouseover="self.status='Returns a report describing PCR primer properties'; return true;" onmouseout="self.status=' '; return true;">-PCR Primer Stats</a><br />
    <a href="pcr_products.html" onmouseover="self.status='Generates PCR products from a template and two primer sequences.'; return true;" onmouseout="self.status=' '; return true;">-PCR Products</a><br />
    <a href="protein_gravy.html" onmouseover="self.status='Returns the grand average of hydropathy value of protein sequences.'; return true;" onmouseout="self.status=' '; return true;">-Protein GRAVY</a><br />
    <a href="protein_iep.html" onmouseover="self.status='Returns the predicted isoelectric point of protein sequences.'; return true;" onmouseout="self.status=' '; return true;">-Protein Isoelectric Point</a><br />
    <a href="protein_mw.html" onmouseover="self.status='Calculates the molecular weight of protein sequences.'; return true;" onmouseout="self.status=' '; return true;">-Protein Molecular Weight</a><br />
    <a href="protein_pattern.html" onmouseover="self.status='Returns positions of the patterns you enter.'; return true;" onmouseout="self.status=' '; return true;">-Protein Pattern Find</a><br />
    <a href="protein_stats.html" onmouseover="self.status='Returns basic sequence statistics.'; return true;" onmouseout="self.status=' '; return true;">-Protein Stats</a><br />
    <a href="rest_digest.html" onmouseover="self.status='Converts the sequence you enter into restriction fragments.'; return true;" onmouseout="self.status=' '; return true;">-Restriction Digest</a><br />
    <a href="rest_summary.html" onmouseover="self.status='Returns the number and positions of restriction sites.'; return true;" onmouseout="self.status=' '; return true;">-Restriction Summary</a><br />
    <a href="rev_trans.html" onmouseover="self.status='Can be used to convert protein into DNA.'; return true;" onmouseout="self.status=' '; return true;">-Reverse Translate</a><br />
    <a href="translate.html" onmouseover="self.status='Returns the translation in the reading frame you specify.'; return true;" onmouseout="self.status=' '; return true;">-Translate</a>
  </div>

 <div class="category">
    Random Sequences
  </div>

  <div class="program">
    <a href="mutate_dna.html" onmouseover="self.status='Introduces random mutations into DNA sequences.'; return true;" onmouseout="self.status=' '; return true;">-Mutate DNA</a><br />
    <a href="mutate_protein.html" onmouseover="self.status='Introduces random mutations into protein sequences.'; return true;" onmouseout="self.status=' '; return true;">-Mutate Protein</a><br />
    <a href="random_coding_dna.html" onmouseover="self.status='Generates a random coding sequence of the length you specify.'; return true;" onmouseout="self.status=' '; return true;">-Random Coding DNA</a><br />
    <a href="random_dna.html" onmouseover="self.status='Generates a random DNA sequence of the length you specify.'; return true;" onmouseout="self.status=' '; return true;">-Random DNA Sequence</a><br />
    <a href="random_dna_regions.html" onmouseover="self.status='Replaces regions of the DNA sequences you enter with random bases.'; return true;" onmouseout="self.status=' '; return true;">-Random DNA Regions</a><br />
    <a href="random_protein.html" onmouseover="self.status='Generates a random protein sequence of the length you specify.'; return true;" onmouseout="self.status=' '; return true;">-Random Protein Sequence</a><br />
    <a href="random_protein_regions.html" onmouseover="self.status='Replaces regions of the protein sequences you enter with random residues.'; return true;" onmouseout="self.status=' '; return true;">-Random Protein Regions</a><br />
    <a href="sample_dna.html" onmouseover="self.status='Samples bases from a DNA sequence with replacement.'; return true;" onmouseout="self.status=' '; return true;">-Sample DNA</a><br />
    <a href="sample_protein.html" onmouseover="self.status='Samples residues from a protein sequence with replacement.'; return true;" onmouseout="self.status=' '; return true;">-Sample Protein</a><br />
    <a href="shuffle_dna.html" onmouseover="self.status='Randomly shuffles the DNA sequences you enter.'; return true;" onmouseout="self.status=' '; return true;">-Shuffle DNA</a><br />
    <a href="shuffle_protein.html" onmouseover="self.status='Randomly shuffles the protein sequences you enter.'; return true;" onmouseout="self.status=' '; return true;">-Shuffle Protein</a>
  </div>

  <div class="category">
    Miscellaneous
  </div>

  <div class="program">
    <a href="iupac.html" onmouseover="self.status='IUPAC codes for DNA and protein.'; return true;" onmouseout="self.status=' '; return true;">-IUPAC codes</a><br />
    <a href="genetic_code.html" onmouseover="self.status='The genetic codes used in the Sequence Manipulation Suite.'; return true;" onmouseout="self.status=' '; return true;">-Genetic codes</a><br />
    <a href="about.html" onmouseover="self.status='About the Sequence Manipulation Suite.'; return true;" onmouseout="self.status=' '; return true;">-About this site</a><br />
  </div>
<!-- END INSERT LINKS -->

          <div class="copyright">
            <br />
            Biological Sequence Analyisis Manipulation Suite  &copy; 2016 Pazhani Ragunathan.
          </div>

          <center>
            <div class="program">
              <a href="index.html" target="_top">home</a>
            </div>
          </center>
        </td><!-- PROGRAM CONTENT -->

        <td class="program">
          <table border="0" cellspacing="0" cellpadding="2">
            <tbody>
              <tr>
                <td class="sms">Biological Sequence Analyisis Manipulation Suite:</td>
              </tr>

              <tr>
                <td class="title">About</td>
              </tr>

              <tr>
                <td class="description">
                  Biological Sequence Analyisis Manipulation Suite is written in JavaScript, which is a lightweight, cross-platform, object-oriented scripting language. JavaScript is now standardized by the ECMA (European Computer Manufacturers Association). The first version of the ECMA standard is documented in the ECMA-262 specification. The ECMA-262 standard is also approved by the ISO (International Organization for Standards) as ISO-16262. JavaScript 1.5 is fully compatible with ECMA-262, Edition 3.<br />
                  <br />
                  Sequences submitted to the Sequence Manipulation Suite do not leave your computer and are instead manipulated by your web browser, which executes the JavaScript.</a>.<br />
                  <br />
                  Here are short descriptions of the programs that comprise the Sequence Manipulation Suite:<br />
                  <br />

                  </ul><b>Sequence Analysis:</b>

                  <ul>
                    <li><b>Codon Plot</b> - accepts a DNA sequence and generates a graphical plot consisting of a horizontal bar for each codon. The length of the bar is proportional to the frequency of the codon in the codon frequency table you enter. Use Codon Plot to find portions of DNA sequence that may be poorly expressed, or to view a graphic representation of a codon usage table (by using a DNA sequence consisting of one of each codon type).</li>

                    <li><b>Codon Usage</b> - accepts one or more DNA sequences and returns the number and frequency of each codon type. Since the program also compares the frequencies of codons that code for the same amino acid (synonymous codons), you can use it to assess whether a sequence shows a preference for particular synonymous codons.</li>

                    <li><b>CpG Islands</b> - reports potential CpG island regions using the method described by <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=3656447&amp;dopt=Abstract" target="_blank">Gardiner-Garden and Frommer (1987)</a>. The calculation is performed using a 200 bp window moving across the sequence at 1 bp intervals. CpG islands are defined as sequence ranges where the Obs/Exp value is greater than 0.6 and the GC content is greater than 50%. The expected number of CpG dimers in a window is calculated as the number of 'C's in the window multiplied by the number of 'G's in the window, divided by the window length. CpG islands are often found in the 5' regions of vertebrate genes, therefore this program can be used to highlight potential genes in genomic sequences.</li>

                    <li><b>DNA Molecular Weight</b> - accepts one or more DNA sequences and calculates molecular weight. Sequences can be treated as double-stranded or single-stranded, and as linear or circular. Use DNA Molecular Weight when calculating molecule copy number.</li>

                    <li><b>DNA Pattern Find</b> - accepts one or more sequences along with a search pattern and returns the number and positions of sites that match the pattern. The search pattern is written as a JavaScript regular expression, which resembles the regular expressions written in other programming languages, such as Perl.</li>

                    <li><b>DNA Stats</b> - returns the number of occurrences of each residue in the sequence you enter. Percentage totals are also given for each residue, and for certain groups of residues, allowing you to quickly compare the results obtained for different sequences.</li>

                    <li><b>Fuzzy Search DNA</b> - accepts a DNA sequence along with a query sequence and returns sites that are identical or similar to the query. You can use this program, for example, to find sequences that can be easily mutated into a useful restriction site.</li>

                    <li><b>Fuzzy Search Protein</b> - accepts a protein sequence along with a query sequence and returns sites that are identical or similar to the query.</li>

                    <li><b>Ident and Sim</b> - accepts a group of aligned sequences (in FASTA or GDE format) and calculates the identity and similarity of each sequence pair. Identity and similarity values are often used to assess whether or not two sequences share a common ancestor or function.</li>

                    <li><b>Mutate for Digest</b> - accepts a DNA sequence as input and searches for regions that can easily be mutated to create a restriction site of interest. The program also reports protein translations so that you can see which reading frames are altered by the proposed mutations. Use Mutate for Digest to find sequences that can be converted to a useful restriction site using PCR or site-directed mutagenesis.</li>

                    <li><b>Multi Rev Trans</b> - accepts a protein alignment and uses a codon usage table to generate a degenerate DNA coding sequence. The program also returns a graph that can be used to find regions of minimal degeneracy at the nucleotide level. Use Multi Rev Trans when designing PCR primers to anneal to an unsequenced coding sequence from a related species.</li>

                    <li><b>ORF Finder</b> - searches for open reading frames (ORFs) in the DNA sequence you enter. The program returns the range of each ORF, along with its protein translation. ORF Finder supports the entire IUPAC alphabet and several genetic codes. Use ORF Finder to search newly sequenced DNA for potential protein encoding segments.</li>

                    <li><b>Pairwise Align Codons</b> - accepts two coding sequences and determines the optimal global alignment. Use Pairwise Align Codons to look for conserved coding sequence regions.</li>

                    <li><b>Pairwise Align DNA</b> - accepts two DNA sequences and determines the optimal global alignment. Use Pairwise Align DNA to look for conserved sequence regions.</li>

                    <li><b>Pairwise Align Protein</b> - accepts two protein sequences and determines the optimal global alignment. Use Pairwise Align Protein to look for conserved sequence regions.</li>

                    <li><b>PCR Primer Stats</b> - accepts a list of PCR primer sequences and returns a report describing the properties of each primer, including melting temperature, percent GC content, and PCR suitability. Use PCR Primer Stats to evaluate potential PCR primers.</li>

                    <li><b>PCR Products</b> - accepts one or more DNA sequence templates and two primer sequences. The program searches for perfectly matching primer annealing sites that can generate a PCR product. Any resulting products are sorted by size, and they are given a title specifying their length, their position in the original sequence, and the primers that produced them. You can use linear or circular molecules as the template. Use PCR Products to determine the product sizes you can expect to see when you perform PCR in the lab.</li>

                    <li><b>Protein GRAVY</b> - Protein GRAVY returns the GRAVY (grand average of hydropathy) value for the protein sequences you enter. The GRAVY value is calculated by adding the hydropathy value for each residue and dividing by the length of the sequence (<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7108955&amp;dopt=Abstract" target="_blank">Kyte and Doolittle; 1982</a>).</li>

                    <li><b>Protein Isoelectric Point</b> - calculates the theoretical pI (isoelectric point) for the protein sequence you enter. Use Protein Isoelectric Point when you want to know approximately where on a 2-D gel a particular protein will be found.</li>

                    <li><b>Protein Molecular Weight</b> - accepts one or more protein sequences and calculates molecular weight. You can append copies of commonly used epitopes and fusion proteins using the supplied list. Use Protein Molecular Weight when you wish to predict the location of a protein of interest on a gel in relation to a set of protein standards.</li>

                    <li><b>Protein Pattern Find</b> - accepts one or more sequences along with a search pattern and returns the number and positions of sites that match the pattern. The search pattern is written as a JavaScript regular expression, which resembles the regular expressions written in other programming languages, such as Perl.</li>

                    <li><b>Protein Stats</b> - returns the number of occurrences of each residue in the sequence you enter. Percentage totals are also given for each residue, and for certain groups of residues, allowing you to quickly compare the results obtained for different sequences.</li>

                    <li><b>Restriction Digest</b> - cleaves a DNA sequence in a virtual restriction digest, with one, two, or three restriction enzymes. The resulting fragments are sorted by size, and they are given a title specifying their length, their position in the original sequence, and the enzyme sites that produced them. You can digest linear or circular molecules, and even a mixture of molecules (by entering more than one sequence in FASTA format). Use Restriction Digest to determine the fragment sizes you will see when you perform a digest in the lab.</li>

                    <li><b>Restriction Summary</b> - accepts a DNA sequence and returns the number and positions of commonly used restriction endonuclease cut sites. Use this program if you wish to quickly determine whether or not an enzyme cuts a particular segment of DNA.</li>

                    <li><b>Reverse Translate</b> - accepts a protein sequence as input and uses a codon usage table to generate a DNA sequence representing the most likely non-degenerate coding sequence. A consensus sequence derived from all the possible codons for each amino acid is also returned. Use Reverse Translate when designing PCR primers to anneal to an unsequenced coding sequence from a related species.</li>

                    <li><b>Translate</b> - accepts a DNA sequence and converts it into a protein in the reading frame you specify. Translate supports the entire IUPAC alphabet and several genetic codes.</li>

                  </ul><b>Random Sequences:</b>

                  <ul>
                    <li><b>Mutate DNA</b> - introduces base changes into a DNA sequence. You can select the number of mutations to introduce, and whether or not to preserve the first and last three bases in the sequence, to reflect selection acting to maintain start and stop codons. The position of each mutation is chosen randomly, and multiple mutations can occur at a single site. Mutated sequences can be used to evaluate the significance of sequence analysis results.</li>

                    <li><b>Mutate Protein</b> - introduces residue changes into a protein sequence. You can select the number of mutations to introduce, and whether or not to preserve the first residue in the sequence, to reflect selection acting to maintain a start codon. The position of each mutation is chosen randomly, and multiple mutations can occur at a single site. Mutated sequences can be used to evaluate the significance of sequence analysis results.</li>

                    <li><b>Random Coding DNA</b> - generates a random open reading frame beginning with a start codon and ending with a stop codon. You can choose the genetic code to use and the length of the sequence to generate. Random sequences can be used to evaluate the significance of sequence analysis results.</li>

                    <li><b>Random DNA Sequence</b> - generates random sequences of the length you specify. Random sequences can be used to evaluate the significance of sequence analysis results.</li>

                    <li><b>Random DNA Regions</b> - replaces regions of DNA sequences with random bases. Random sequences can be used to evaluate the significance of sequence analysis results.</li>

                    <li><b>Random Protein Sequence</b> - generates random sequences of the length you specify. Random sequences can be used to evaluate the significance of sequence analysis results.</li>

                    <li><b>Random Protein Regions</b> - replaces regions of protein sequences with random residues. Random sequences can be used to evaluate the significance of sequence analysis results.</li>

                    <li><b>Sample DNA</b> - randomly selects bases from the guide sequence until a sequence of the length you specify is constructed. Each selected base is replaced so that it can be selected again.</li>

                    <li><b>Sample Protein</b> - randomly selects bases from the guide sequence until a sequence of the length you specify is constructed. Each selected residue is replaced so that it can be selected again.</li>

                    <li><b>Shuffle DNA</b> - randomly shuffles a DNA sequence. Shuffled sequences can be used to evaluate the significance of sequence analysis results, particularly when sequence composition is an important consideration.</li>

                    <li><b>Shuffle Protein</b> - randomly shuffles a protein sequence. Shuffled sequences can be used to evaluate the significance of sequence analysis results, particularly when sequence composition is an important consideration.</li>
                  </ul><br />

                  <center>
                    <a href="about.html" target="_blank">new window</a> | <a href="index.html">home</a> | <a href="reference.html">citation</a>
                  </center><span class="version">Mon Nov 31 22:17:06 2016</span>
                </td>
              </tr>

              <tr>
                <td><span class="valid">Valid XHTML 1.0; Valid CSS</span></td>
              </tr>
            </tbody>
          </table><!-- MAIN TABLE END -->
        </td>
      </tr>
    </tbody>
  </table>
</body>
</html>
