<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
        "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">

<html lang="en" xmlns="http://www.w3.org/1999/xhtml" xml:lang="en">
<head>
  <meta name="generator" content="WebStrom v2016.3.1 for Linux, see www.w3.org" />

  <title>Reverse Translate</title>
  <meta http-equiv="Content-Type" content="text/html; charset=us-ascii" />
  <link rel="stylesheet" href="includes/stylesheet.css" type="text/css" />
  <script type="text/javascript" src="scripts/sms_common.js">
</script>
  <script type="text/javascript" src="scripts/rev_trans.js">
</script>
</head>

<body class="main" onload="document.main_form.main_submit.focus()">
  <!-- MAIN TABLE -->

  <table border="0" cellspacing="2" cellpadding="2">
    <tbody>
      <tr>
        <td class="links">
          <a href="index.html"><img src="includes/logo.png" alt="SMS logo" width="144" height="56" border="0" /></a> <!-- INSERT LINKS -->

  <div class="category">
    Sequence Analysis
  </div>

  <div class="program">
    <a href="codon_plot.html" onmouseover="self.status='Plots codon frequency (according to the codon table you enter) for each codon in a DNA sequence.'; return true;" onmouseout="self.status=' '; return true;">-Codon Plot</a><br />
    <a href="codon_usage.html" onmouseover="self.status='Returns a standard codon usage table.'; return true;" onmouseout="self.status=' '; return true;">-Codon Usage</a><br />
    <a href="cpg_islands.html" onmouseover="self.status='Returns a list of potential CpG islands.'; return true;" onmouseout="self.status=' '; return true;">-CpG Islands</a><br />
    <a href="dna_mw.html" onmouseover="self.status='Calculates the molecular weight of DNA sequences.'; return true;" onmouseout="self.status=' '; return true;">-DNA Molecular Weight</a><br />
    <a href="dna_pattern.html" onmouseover="self.status='Returns positions of the patterns you enter.'; return true;" onmouseout="self.status=' '; return true;">-DNA Pattern Find</a><br />
    <a href="dna_stats.html" onmouseover="self.status='Returns basic sequence statistics.'; return true;" onmouseout="self.status=' '; return true;">-DNA Stats</a><br />
    <a href="fuzzy_search_dna.html" onmouseover="self.status='Returns sequences that are identical or similar to a query sequence.'; return true;" onmouseout="self.status=' '; return true;">-Fuzzy Search DNA</a><br />
    <a href="fuzzy_search_protein.html" onmouseover="self.status='Returns sequences that are identical or similar to a query sequence.'; return true;" onmouseout="self.status=' '; return true;">-Fuzzy Search Protein</a><br />
    <a href="ident_sim.html" onmouseover="self.status='Accepts aligned sequences in FASTA format and calculates the identity and similarity of each sequence pair.'; return true;" onmouseout="self.status=' '; return true;">-Ident and Sim</a><br />
    <a href="multi_rev_trans.html" onmouseover="self.status='Can be used to predict a DNA sequence in another species using a protein sequence alignment.'; return true;" onmouseout="self.status=' '; return true;">-Multi Rev Trans</a><br />
    <a href="mutate_for_digest.html" onmouseover="self.status='Finds DNA sequences that can easily be converted to a restriction site.'; return true;" onmouseout="self.status=' '; return true;">-Mutate for Digest</a><br />
    <a href="orf_find.html" onmouseover="self.status='Determines the positions of open reading frames.'; return true;" onmouseout="self.status=' '; return true;">-ORF Finder</a><br />
    <a href="pairwise_align_codons.html" onmouseover="self.status='Returns the optimal global alignment for two coding DNA sequences.'; return true;" onmouseout="self.status=' '; return true;">-Pairwise Align Codons</a><br />
    <a href="pairwise_align_dna.html" onmouseover="self.status='Returns the optimal global alignment for two DNA sequences.'; return true;" onmouseout="self.status=' '; return true;">-Pairwise Align DNA</a><br />
    <a href="pairwise_align_protein.html" onmouseover="self.status='Returns the optimal global alignment for two protein sequences.'; return true;" onmouseout="self.status=' '; return true;">-Pairwise Align Protein</a><br />
    <a href="pcr_primer_stats.html" onmouseover="self.status='Returns a report describing PCR primer properties'; return true;" onmouseout="self.status=' '; return true;">-PCR Primer Stats</a><br />
    <a href="pcr_products.html" onmouseover="self.status='Generates PCR products from a template and two primer sequences.'; return true;" onmouseout="self.status=' '; return true;">-PCR Products</a><br />
    <a href="protein_gravy.html" onmouseover="self.status='Returns the grand average of hydropathy value of protein sequences.'; return true;" onmouseout="self.status=' '; return true;">-Protein GRAVY</a><br />
    <a href="protein_iep.html" onmouseover="self.status='Returns the predicted isoelectric point of protein sequences.'; return true;" onmouseout="self.status=' '; return true;">-Protein Isoelectric Point</a><br />
    <a href="protein_mw.html" onmouseover="self.status='Calculates the molecular weight of protein sequences.'; return true;" onmouseout="self.status=' '; return true;">-Protein Molecular Weight</a><br />
    <a href="protein_pattern.html" onmouseover="self.status='Returns positions of the patterns you enter.'; return true;" onmouseout="self.status=' '; return true;">-Protein Pattern Find</a><br />
    <a href="protein_stats.html" onmouseover="self.status='Returns basic sequence statistics.'; return true;" onmouseout="self.status=' '; return true;">-Protein Stats</a><br />
    <a href="rest_digest.html" onmouseover="self.status='Converts the sequence you enter into restriction fragments.'; return true;" onmouseout="self.status=' '; return true;">-Restriction Digest</a><br />
    <a href="rest_summary.html" onmouseover="self.status='Returns the number and positions of restriction sites.'; return true;" onmouseout="self.status=' '; return true;">-Restriction Summary</a><br />
    <a href="rev_trans.html" onmouseover="self.status='Can be used to convert protein into DNA.'; return true;" onmouseout="self.status=' '; return true;">-Reverse Translate</a><br />
    <a href="translate.html" onmouseover="self.status='Returns the translation in the reading frame you specify.'; return true;" onmouseout="self.status=' '; return true;">-Translate</a>
  </div>

  <div class="category">
    Miscellaneous
  </div>

  <div class="program">
    <a href="iupac.html" onmouseover="self.status='IUPAC codes for DNA and protein.'; return true;" onmouseout="self.status=' '; return true;">-IUPAC codes</a><br />
    <a href="genetic_code.html" onmouseover="self.status='The genetic codes used in the Sequence Manipulation Suite.'; return true;" onmouseout="self.status=' '; return true;">-Genetic codes</a><br />
    <a href="about.html" onmouseover="self.status='About the Sequence Manipulation Suite.'; return true;" onmouseout="self.status=' '; return true;">-About this site</a><br />
  </div>
<!-- END INSERT LINKS -->

          <div class="copyright">
            <br />
            Biological Sequence Analyisis Manipulation Suite  &copy; 2016 Pazhani Ragunathan.
          </div>

          <center>
            <div class="program">
              <a href="index.html" target="_top">home</a>
            </div>
          </center>
        </td><!-- PROGRAM CONTENT -->

        <td class="program">
          <table border="0" cellspacing="0" cellpadding="2">
            <tbody>
              <tr>
                <td class="sms">Biological Sequence Manipulation Suite:</td>
              </tr>

              <tr>
                <td class="title">Reverse Translate</td>
              </tr>

              <tr>
                <td class="description">
                  Reverse Translate accepts a protein sequence as input and uses a codon usage table to generate a DNA sequence representing the most likely non-degenerate coding sequence. A consensus sequence derived from all the possible codons for each amino acid is also returned. Use Reverse Translate when designing PCR primers to anneal to an unsequenced coding sequence from a related species.<br />
                  <br />

                  <form action="" name="main_form" id="main_form">
                    Paste the raw sequence or one or more FASTA sequences into the text area below. Input limit is 20000 characters.<br />
                    <textarea rows="6" cols="61">
&gt;sample sequence
ACDEFGHIKLMNPQRSTVWY*
</textarea><br />

                    <input type="button" value="Submit" name="main_submit" onclick="try{revTrans(document)} catch(e) {alert('The following error was encountered: ' + e);}" /> <input type="button" value="Clear" onclick="document.forms[0].elements[0].value = ' '; document.forms[0].elements[4].value = ' '" /> <input type="reset" value="Reset" /> Enter the codon table you wish to use (in GCG format). The default codon usage table was generated using all the E. coli coding sequences in GenBank. It was obtained from the <a href="http://www.kazusa.or.jp/codon/" target="_blank">Codon Usage Database.</a><br />
                    <textarea rows="6" cols="61">
AmAcid  Codon      Number    /1000     Fraction   ..
 
Gly     GGG     50527.00     11.12      0.15
Gly     GGA     39036.00      8.59      0.12
Gly     GGT    114185.00     25.14      0.34
Gly     GGC    130043.00     28.63      0.39
 
Glu     GAG     83804.00     18.45      0.32
Glu     GAA    179460.00     39.51      0.68
Asp     GAT    146794.00     32.32      0.63
Asp     GAC     87759.00     19.32      0.37
 
Val     GTG    115687.00     25.47      0.36
Val     GTA     51020.00     11.23      0.16
Val     GTT     86572.00     19.06      0.27
Val     GTC     67356.00     14.83      0.21
 
Ala     GCG    146264.00     32.20      0.34
Ala     GCA     93390.00     20.56      0.22
Ala     GCT     73677.00     16.22      0.17
Ala     GCC    113412.00     24.97      0.27
 
Arg     AGG      7423.00      1.63      0.03
Arg     AGA     12345.00      2.72      0.05
Ser     AGT     41544.00      9.15      0.15
Ser     AGC     70867.00     15.60      0.26
 
Lys     AAG     51685.00     11.38      0.25
Lys     AAA    156169.00     34.38      0.75
Asn     AAT     84846.00     18.68      0.46
Asn     AAC     98018.00     21.58      0.54
 
Met     ATG    123604.00     27.21      1.00
Ile     ATA     24233.00      5.34      0.09
Ile     ATT    135873.00     29.92      0.50
Ile     ATC    111878.00     24.63      0.41
 
Thr     ACG     63696.00     14.02      0.26
Thr     ACA     35995.00      7.93      0.15
Thr     ACT     43256.00      9.52      0.18
Thr     ACC    103121.00     22.70      0.42
 
Trp     TGG     65630.00     14.45      1.00
End     TGA      4428.00      0.97      0.30
Cys     TGT     23461.00      5.17      0.45
Cys     TGC     28747.00      6.33      0.55
 
End     TAG      1172.00      0.26      0.08
End     TAA      9006.00      1.98      0.62
Tyr     TAT     75774.00     16.68      0.58
Tyr     TAC     55847.00     12.30      0.42
 
Leu     TTG     60322.00     13.28      0.13
Leu     TTA     62823.00     13.83      0.13
Phe     TTT    100128.00     22.05      0.57
Phe     TTC     74885.00     16.49      0.43
 
Ser     TCG     39546.00      8.71      0.15
Ser     TCA     35837.00      7.89      0.13
Ser     TCT     42367.00      9.33      0.16
Ser     TCC     40365.00      8.89      0.15
 
Arg     CGG     25751.00      5.67      0.10
Arg     CGA     16607.00      3.66      0.07
Arg     CGT     93997.00     20.70      0.37
Arg     CGC     96053.00     21.15      0.38
 
Gln     CAG    130898.00     28.82      0.66
Gln     CAA     67129.00     14.78      0.34
His     CAT     57585.00     12.68      0.57
His     CAC     43743.00      9.63      0.43
 
Leu     CTG    231373.00     50.94      0.49
Leu     CTA     18067.00      3.98      0.04
Leu     CTT     51442.00     11.33      0.11
Leu     CTC     48147.00     10.60      0.10
 
Pro     CCG    101467.00     22.34      0.51
Pro     CCA     38663.00      8.51      0.20
Pro     CCT     32678.00      7.19      0.17
Pro     CCC     24383.00      5.37      0.12
</textarea><br />
                  </form>*This page requires JavaScript.<br />
                  <br />

                  <center>
                    <a href="rev_trans.html" target="_blank">new window</a> | <a href="index.html">home</a>
                  </center><span class="version">Mon Nov 31 22:17:06 2016</span>
                </td>
              </tr>

              <tr>
                <td><span class="valid">Valid XHTML 1.0; Valid CSS</span></td>
              </tr>
            </tbody>
          </table><!-- MAIN TABLE END -->
        </td>
      </tr>
    </tbody>
  </table>
</body>
</html>
